apps/src/contrast.cpp

```diff
@@ -43,8 +98,15 @@ int main(int argc, char** argv)
     inkpts.push_back({577, 1722});
     inkpts.push_back({627, 1712});
     inkpts.push_back({769, 1682});
+    */
+    bool inkptsEmpty = True;
```

Comment:
After you add the `doMichelson` bool up above, wrap loading the pts files in a `if(doMichelson)` block. Let's you get rid of inkPtsEmpty and papyPtsEmpty.